docs/README.md

```diff
@@ -141,15 +139,15 @@ To interact with a measurement in InstrumentStudio, follow the steps below:
 
 3. The layout will be populated with the UI created for the measurement, containing the provided parameters.
 
-4. Provide the required configurations and click `Run` to run the measurement.
+4. Provide the required configurations and click `RUN` to run the measurement.
 
     ![Running the measurement](images/Running%20the%20measurement.png)
 
 ---
 
 ## Creating a LabVIEW executable for static registration
 
-To create an executable for the measurement service project follow the steps provided below,
+A Build Specification is included when the measurement service is created with the generator. To create an executable for the measurement service, simply build the build specification. The following steps are all pre-configured for the build specification, but they show the steps you would need to perform to create your own build spec and executable.
```

Comment:
I support updating this text to note that the generator will provide the customer with a build specification.  Since the generator does that for them, do we need subsequent steps to show them how to replicate it themselves anymore?  I'm presuming that this documentation pre-dates the generated build spec, thus it was more necessary at the time.

Reply:
I was thinking the same thing. I would be okay with removing this documentation. I think using the generator is pretty much a requirement to writing a measurement.

Reply:
Chopped all those instructions and updated the paragraph about the build spec we include.